projects/storefrontlib/src/cms-components/organization/units/unit-tree-navigation-ui/unit-tree-navigation-ui.component.spec.ts

```diff
@@ -0,0 +1,212 @@
+import { Component, CUSTOM_ELEMENTS_SCHEMA, Input, DebugElement } from '@angular/core';
+import { TestBed, ComponentFixture, async } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { UnitTreeNavigationUIComponent } from './unit-tree-navigation-ui.component';
+import { BreakpointService } from '../../../../layout/breakpoint/breakpoint.service';
+import { BREAKPOINT } from '../../../../layout/config/layout-config';
+import { of, Observable } from 'rxjs';
+import { map } from 'rxjs/internal/operators/map';
+import { I18nTestingModule } from '@spartacus/core';
+import { NavigationNode } from '../../../navigation/navigation/navigation-node.model';
+
+@Component({
+  template: '',
+  selector: 'cx-unit-tree-navigation-ui',
+})
+class MockNavigationComponent {
+  @Input() node: NavigationNode;
+  selectedNode: NavigationNode;
+  @Input() defaultExpandLevel: number;
+}
+
+class MockBreakpointService {
+  get breakpoint$(): Observable<BREAKPOINT> {
+    return of();
+  }
+  get breakpoints(): BREAKPOINT[] {
+    return [
+      BREAKPOINT.xs,
+      BREAKPOINT.sm,
+      BREAKPOINT.md,
+      BREAKPOINT.lg,
+      BREAKPOINT.xl,
+    ];
+  }
+  isDown(breakpoint: BREAKPOINT) {
+    return this.breakpoint$.pipe(
+      map((br) =>
+        this.breakpoints
+          .slice(0, this.breakpoints.indexOf(breakpoint) + 1)
+          .includes(br)
+      )
+    );
+  }
+}
+
+const mockNode: NavigationNode = {
+  title: 'test',
+  children: [
+    {
+      title: 'Root 1',
+      url: '/root-1',
+      children: [
+        {
+          title: 'Child 1',
+          url: "/child-1",
+          children: [
+            {
+              title: 'Sub child 1',
+              children: [
+                {
+                  title: 'Sub sub child 1',
+                  url: '/sub-sub-child-1',
+                },
+                {
+                  title: 'Sub sub child 2',
+                  url: '/sub-sub-child-2',
+                },
+                {
+                  title: 'Sub sub child 3',
+                  url: '/sub-sub-child-3',
+                },
+                {
+                  title: 'Sub sub child 4',
+                  url: '/sub-sub-child-4',
+                },
+              ],
+            },
+          ],
+        },
+        {
+          title: 'Child 2',
+          url: '/child-2',
+        },
+      ],
+    },
+    {
+      title: 'Root 2',
+      url: '/root-2',
+    },
+  ],
+};
+
+fdescribe('UnitTreeNavigationUIComponent', () => {
```

Comment:
It should be `describe`.

Reply:
We are using `fdescribe` only locally to prevent reloading full sets of tests every time.